scipy/ndimage/measurements.py

```diff
@@ -264,10 +265,13 @@ def find_objects(input, max_label=0):
     if numpy.iscomplexobj(input):
         raise TypeError('Complex type not supported')
 
+    if input.dtype == np.bool:
+        input = input.astype(int)
```

Comment:
Wouldn't `input = input.view(np.uint8)` work too? It would save a copy of the array.